packages/x-components/src/x-modules/facets/components/lists/__tests__/selected-filters-list.spec.ts

```diff
@@ -100,19 +104,24 @@ describe('testing SelectedFiltersList component', () => {
         <SelectedFiltersList>
           <template #default="{ filter }">{{ filter.label }} selected!</template>
           <template #brand="{ filter }">Which one? {{ filter.label }}</template>
+          <template #root-categories="{ filter }">Much better: {{ filter.label }}</template>
+          <template #rootCategories="{ filter }">Much better: {{ filter.label }}</template>
```

Comment:
What are you testing with this? I see no assertions related with this line. It is exactly the same than the one above, so you will never know if it is the `root-categories` slot or the `rootCategories` slot the one it is being rendered.

 If you change the text of the `rootCategories` slot to something like `Shouldn't be rendered: {{ filter.label }}`, it is enough, because you are checking that exactly 3 filters are rendered with a different text each one.